src/Tribe/Aggregator/Tabs/New.php

```diff
@@ -284,6 +284,14 @@ public function get_result_messages( $queue ) {
 				);
 			}
 
+			$images_scheduled = $queue->activity->get( 'images', 'scheduled' );
+			if ( ! empty( $images_scheduled ) ) {
+				$messages['success'][] = sprintf( // add image import count
+						_n( '%1$d new image was scheduled for import.', '%1$d new images were scheduled for import.', $queue->activity->count( 'images', 'scheduled' ), 'the-events-calendar' ),
```

Comment:
* :no_entry_sign: Multi-line function call not indented correctly; expected 20 spaces but found 24
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: aabf13206ec43ac2dcdc830dbf684559</sub>